### ISSUE TITLE
Track Status / Support Life Cycle of packages (supported/unsupported/abondoned/...)

### ISSUE THREAD

**Issue by valentijnscholten** [DependencyTrack/dependency-track#4476]
### Current Behavior

Status / Support Life Cycle of packages is not tracked by Dependency Track

### Proposed Behavior

Some package repositories provide information about packages that are abondoned (Composer). Other repositories might also provide information about whether packages are supported or unsupported.

It would be good if these could be tracked by Dependency Track, for example to add risk scores or flags to packages that are abandoned.

Some example from Compose / Packagist:

https://packagist.org/p2/laminas/laminas-text.json
`"abandoned": true`

https://packagist.org/p2/laminas/laminas-mail.json

`"abandoned": "symfony/mailer"`

`symfony-mailer` is suggested as replacement for the abandoned `laminas-mail`.

### Checklist

- [x] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#filing-issues)
- [x] I have checked the [existing issues](https://github.com/DependencyTrack/dependency-track/issues) for whether this enhancement was already requested

**Comment by mieliespoor** [comment]
Yes, nuget has the ability to deprecate packages. It would be good if that could be flagged as an operational risk.
